src/lib/y2storage/boot_requirements_strategies/zipl.rb

```diff
@@ -48,12 +64,34 @@ def zipl_partition_missing?
         free_mountpoint?("/boot/zipl")
       end
 
+      # @return [VolumeSpecification]
+      def zipl_volume
+        return @zipl_volume unless @zipl_volume.nil?
+
+        @zipl_volume = VolumeSpecification.new({})
+        @zipl_volume.mount_point = "/boot/zipl"
+        @zipl_volume.fs_types = [Filesystems::Type::EXT2]
```

Comment:
I guess any EXT{2..4} is supported here. Somebody should know :-)

Reply:
I seems there is no guarantee about that.